.travis.yml

```diff
@@ -29,4 +40,6 @@ after_deploy:
 - 'curl -s -X POST -H "Content-Type: application/json" -H "Accept: application/json" -H "Travis-API-Version: 3" -H "Authorization: token $TRAVIS_TOKEN" -d "{\"request\":{\"message\": \"Triggered by kiali-ui change: $TRAVIS_COMMIT\", \"branch\":\"master\"}}" https://api.travis-ci.org/repo/kiali%2Fkiali/requests'
 env:
   global:
-    secure: aI1uMj0h6xjzp2Xl8VeG4Eibnmz77OaTs8vYyOwgf4L5eN59Rz21oJE4j4AFkDBELUJvYHJix/EKWoIGszDFN/XCoOcB39h9MFyrHT74dwmjH1CHDg97e8KZU8oKpvpJ6D5c1tYUrbuzeK+pxqGptMi3zi9Cx8UpN9CavVa1QkcfEf0VfVdGpHIf+EbP9imEnArfPkUMfx0Schp761gySOvgtzzjIUwFLrrszSWM5s5KA8Z2BWMKN4877uey2HCYqfju9TrjWv396QxWxdnzqGvTQCO5jabcxSuK3N5IC76mlkuT/w0rxHVwULDU9JyBQo6gYGkHavKq5vvTOrWzEZwhL5mY6RvCwPg4GJma+O6DWZQRLEbF18Ubu2pm8BbLp+EUdgQMN+rBhTcbhX/om+tZkyODoVmxSEibSo6LcDJ3ZiS8pzlFVQiUbQ3wiQqcIPoDwlhhahwUutd35zt26HJvSs4HuRZ1R6TU3xtsHFVQULbIOMWcWd5dFnzTdBJW3aNu45vkWOE7wtANiAKmpkFowq0lRoKj6GdHgDEk9L0xy/QmoiCmCnrqtSsIill0GjmPpXWzIBQmS4uPUkjGveIqIGfrRqnSyRa6+f0wY4GQ0vkbF4DFbCTvRVcfvvph2R3MS/Tjhvt5L5Bhw52lg/DiWib2Synle1WyIjhXTLk=
+    - secure: aI1uMj0h6xjzp2Xl8VeG4Eibnmz77OaTs8vYyOwgf4L5eN59Rz21oJE4j4AFkDBELUJvYHJix/EKWoIGszDFN/XCoOcB39h9MFyrHT74dwmjH1CHDg97e8KZU8oKpvpJ6D5c1tYUrbuzeK+pxqGptMi3zi9Cx8UpN9CavVa1QkcfEf0VfVdGpHIf+EbP9imEnArfPkUMfx0Schp761gySOvgtzzjIUwFLrrszSWM5s5KA8Z2BWMKN4877uey2HCYqfju9TrjWv396QxWxdnzqGvTQCO5jabcxSuK3N5IC76mlkuT/w0rxHVwULDU9JyBQo6gYGkHavKq5vvTOrWzEZwhL5mY6RvCwPg4GJma+O6DWZQRLEbF18Ubu2pm8BbLp+EUdgQMN+rBhTcbhX/om+tZkyODoVmxSEibSo6LcDJ3ZiS8pzlFVQiUbQ3wiQqcIPoDwlhhahwUutd35zt26HJvSs4HuRZ1R6TU3xtsHFVQULbIOMWcWd5dFnzTdBJW3aNu45vkWOE7wtANiAKmpkFowq0lRoKj6GdHgDEk9L0xy/QmoiCmCnrqtSsIill0GjmPpXWzIBQmS4uPUkjGveIqIGfrRqnSyRa6+f0wY4GQ0vkbF4DFbCTvRVcfvvph2R3MS/Tjhvt5L5Bhw52lg/DiWib2Synle1WyIjhXTLk=
+    - CI=true
```

Comment:
I think this is the default value
 https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

Reply:
Thanks. I'll remove CI=true